src/Common/AsynchronousMetrics.cpp

```diff
@@ -894,6 +897,15 @@ void AsynchronousMetrics::update(TimePoint update_time)
             readText(cgroup_mem_limit_in_bytes, *cgroupmem_limit_in_bytes);
```

Comment:
The variable names:
- cgroupmem_limit_in_bytes
- cgroup_mem_limit_in_bytes

are confusing.

Reply:
true, let me think to refactor

Reply:
refactor to extract shared code of v1 and v2, now there is better var name for file